filebrowser/run.sh

```diff
@@ -1,11 +1,21 @@
 #!/usr/bin/env bash
 
 BOLD='\033[0;1m'
-printf "$${BOLD}Installing filebrowser \n\n"
 
-curl -fsSL https://raw.githubusercontent.com/filebrowser/get/master/get.sh | bash
-
-printf "🥳 Installation complete! \n\n"
+# Check if filebrowser is installed
+if command -v filebrowser &> /dev/null; then
+    printf "🥳 Filebrowser is already installed. Skipping installation.\n\n$"
+else
+    printf "${BOLD}Installing Filebrowser...\n\n"
```

Comment:
```suggestion
    printf "$${BOLD}Installing Filebrowser...\n\n"
```
${} are templates by terraform so we need $$ for shell variables.